src/styles/star.scss

```diff
@@ -0,0 +1,41 @@
+@import "./value.scss";
+
+.card__block-stars {
```

Comment:
Move it index.scss file cause it`s a part of card BEM block and not a separate block. Keep in this file only star block